examples/notebooks_new_field/introduction.ipynb

```diff
@@ -0,0 +1,1099 @@
+    "hz     = (z1-z0)/(nptz-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the spatial parameters are defined, we start the construction of the *spatial grid* using the Devito structure. The required parameters are:\n",
+    "\n",
+    "- *origin* is the point of origin of the grid;\n",
+    "- *extent* is the vector with the lengths of the domain in each of the directions;\n",
+    "- *shape* is the vector with the number of points in each direction;\n",
+    "- *spacing* is the vector with the spacing in each of the directions;\n",
+    "\n",
+    "These parameters are set as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "origin  = (x0,z0)\n",
+    "extent  = (compx,compz)\n",
+    "shape   = (nptx,nptz)\n",
+    "spacing = (hx,hz)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the mesh parameters are built, we can define the structures that represent $\\Omega$ subdomains, that is, particular regions of $\\Omega $, named *subdomains*. In the present notebook, there is no need to split  the domain in particular subregions, so we define a single *subdomain* that correponds to the full domain $\\Omega$. This *subdomain* is built with the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class d0domain(SubDomain):\n",
+    "    name = 'd0'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: z, z: z}\n",
+    "d0_domain = d0domain()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After defining the spatial parameters and *subdomains*, we generate the *spatial grid*:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid = Grid(origin=origin,extent=extent,shape=shape,subdomains=(d0_domain))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The velocity field is set bellow, representing four different layers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v0 = np.zeros((nptx,nptz))                     \n",
+    "p0 = 0    \n",
+    "p1 = int((1/4)*nptz)\n",
+    "p2 = int((1/2)*nptz)\n",
+    "p3 = int((2/3)*nptz)\n",
+    "p4 = nptz  \n",
+    "v0[0:nptx,p0:p1] = 1.5\n",
+    "v0[0:nptx,p1:p2] = 3.0\n",
+    "v0[0:nptx,p2:p3] = 5.0\n",
+    "v0[0:nptx,p3:p4] = 7.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To build the *temporal grid* we use the command *TimeAxis*. Before using  *TimeAxis*, we need to set the following parameters:\n",
+    "\n",
+    "- *t0* is the initial time simulation in milliseconds (ms);\n",
```

Comment:
Start of time simulation

Reply:
The name of this variable is correct if we consider the context of numerical problem.